app/models/host.rb

```diff
@@ -92,14 +92,14 @@ class Host < ApplicationRecord
     profile ||= RequestStore.store['scoped_search_context_profiles']
     profile_ids = profile&.pluck(:id) || []
 
-    sq = Host.left_outer_joins(test_results: :rule_results)
-             .merge(TestResult.latest('LEFT OUTER JOIN'))
+    sq = Host.joins(test_results: :rule_results)
+             .merge(TestResult.latest)
```

Comment:
I adjusted the `latest` to take an optional argument, that can be reverted I guess as it is no longer necessary.